doc/api/n-api.md

```diff
@@ -5189,3 +5180,4 @@ This API may only be called from the main thread.
 [async_hooks `type`]: async_hooks.html#async_hooks_type
 [context-aware addons]: addons.html#addons_context_aware_addons
 [worker threads]: https://nodejs.org/api/worker_threads.html
+[node-addon-api]: https://github.com/nodejs/node-addon-api
```

Comment:
Note that these are in ASCII order, or at least we usually *try* to keep them in ASCII order. So, can you move this one line up, above `worker threads` and below `context-aware addons`?

Reply:
Updated :)